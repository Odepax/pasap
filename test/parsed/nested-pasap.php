<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Pasap Test</title>
	</head>
	<body>
		<h1>The News</h1>
		<news-list>
			<news title="One" author="Pierre-Orélien Mépris">
				<p>
					Lorem ipsum dolor sit amet, consectetur adipisicing elit.
					Atque deserunt dicta dolore et mollitia.
					Accusantium at atque dolor dolore, eligendi eum, facere illum,
					odit omnis placeat quas recusandae sint voluptas.
				</p>
				<p>
					Lorem ipsum dolor sit amet, consectetur adipisicing elit.
					Atque deserunt dicta dolore et mollitia.
					Accusantium at atque dolor dolore, eligendi eum, facere illum,
					odit omnis placeat quas recusandae sint voluptas.
				</p>
			</news>
			<news title="Two" author="Jacques-Pascal Misengarde">
				<p>
					Lorem ipsum dolor sit amet, consectetur adipisicing elit.
					Atque deserunt dicta dolore et mollitia.
					Accusantium at atque dolor dolore, eligendi eum, facere illum,
					odit omnis placeat quas recusandae sint voluptas.
				</p>
				<p>
					Lorem ipsum dolor sit amet, consectetur adipisicing elit.
					Atque deserunt dicta dolore et mollitia.
					Accusantium at atque dolor dolore, eligendi eum, facere illum,
					odit omnis placeat quas recusandae sint voluptas.
				</p>
			</news>
		</news-list>
	</body>
</html>
