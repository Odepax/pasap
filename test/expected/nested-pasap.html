<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Pasap Test</title>
	</head>
	<body>
		<h1>The News</h1>
		<section class="news-list">
			<article class="news">
				<header class="news_meta">
					<h1 class="news_title">One</h1>
					<em>Pierre-Orélien Mépris</em>
				</header>
				<main class="news_content">
					<p>
						Lorem ipsum dolor sit amet, consectetur adipisicing elit.
						Atque deserunt dicta dolore et mollitia.
						Accusantium at atque dolor dolore, eligendi eum, facere illum,
						odit omnis placeat quas recusandae sint voluptas.
					</p>
					<p>
						Lorem ipsum dolor sit amet, consectetur adipisicing elit.
						Atque deserunt dicta dolore et mollitia.
						Accusantium at atque dolor dolore, eligendi eum, facere illum,
						odit omnis placeat quas recusandae sint voluptas.
					</p>
				</main>
			</article>
			<article class="news">
				<header class="news_meta">
					<h1 class="news_title">Two</h1>
					<em>Jacques-Pascal Misengarde</em>
				</header>
				<main class="news_content">
					<p>
						Lorem ipsum dolor sit amet, consectetur adipisicing elit.
						Atque deserunt dicta dolore et mollitia.
						Accusantium at atque dolor dolore, eligendi eum, facere illum,
						odit omnis placeat quas recusandae sint voluptas.
					</p>
					<p>
						Lorem ipsum dolor sit amet, consectetur adipisicing elit.
						Atque deserunt dicta dolore et mollitia.
						Accusantium at atque dolor dolore, eligendi eum, facere illum,
						odit omnis placeat quas recusandae sint voluptas.
					</p>
				</main>
			</article>
		</section>
	</body>
</html>
